packages/apps/dev-wallet/src/pages/backup-recovery-phrase/backup-recovery-phrase.tsx

```diff
@@ -4,10 +4,7 @@ import { MonoDashboardCustomize } from '@kadena/kode-icons/system';
 import { Box, Button, Heading, Stack, Text } from '@kadena/kode-ui';
 import { SideBarBreadcrumbsItem } from '@kadena/kode-ui/patterns';
 import { Link } from 'react-router-dom';
-<SideBarBreadcrumbs icon={<MonoDashboardCustomize />}>
-  <SideBarBreadcrumbsItem href="/">Dashboard</SideBarBreadcrumbsItem>
-  <SideBarBreadcrumbsItem href="/terminal">Dev Console</SideBarBreadcrumbsItem>
-</SideBarBreadcrumbs>;
+
```

Comment:
on a page no breadcrumbs?
if this is not a page,we should move it to components or features folder

Reply:
the page has breadcrumbs these line were outside of the component!, I suppose it was unintentional 
Also I have removed dev console page completely